src/services/db/GitFileSystemService.ts

```diff
@@ -542,10 +542,33 @@ export default class GitFileSystemService {
             isForce
               ? this.git
                   .cwd({ path: `${efsVolPath}/${repoName}`, root: false })
-                  .push(["--force"])
+                  .push([...gitOptions, "--force"])
```

Comment:
why the change here?

Reply:
pushes are supposed to have the required options